augmentedreality/Common/source/ArcGISArViewInterface.cpp

```diff
@@ -22,12 +22,12 @@
 #include <QScreen>
 #include "ArWrapper.h"
 
-
-using namespace Esri::ArcGISRuntime::Toolkit;
 using namespace Esri::ArcGISRuntime::Toolkit::Internal;
 
+namespace Esri::ArcGISRuntime::Toolkit {
+
 /*!
-  \class ArcGISArViewInterface
+  \class Esri::ArcGISRuntime::Toolkit::ArcGISArViewInterface
```

Comment:
Names must be fully qualified in some cases, otherwise qdoc can't find them. Using `namespace` rather than `using namespace`  helps to find declarations.